apps/s_client.c

```diff
@@ -457,7 +453,7 @@ static ossl_ssize_t checked_uint8(const char **inptr, void *out)
 struct tlsa_field {
     void *var;
     const char *name;
-    ossl_ssize_t (*parser)(const char **, void *);
+     ossl_ssize_t(*parser) (const char **, void *);
```

Comment:
This is wrong.  You may need to add a line for `ossl_ssize_t` in `util/indent.pro`...